mesonbuild/cmake/toolchain.py

```diff
@@ -212,7 +212,7 @@ def update_cmake_compiler_state(self) -> None:
         languages = list(self.compilers.keys())
         lang_ids = [language_map.get(x, x.upper()) for x in languages]
         cmake_content = dedent(f'''
-            cmake_minimum_required(VERSION 3.7)
+            cmake_minimum_required(VERSION 3.10)
```

Comment:
Are we able to get away with the same "whatever the current version is" trick here too?